frontend/package.json

```diff
@@ -72,17 +71,18 @@
     "sveltekit-superforms": "^1.13.4",
     "tailwindcss": "catalog:",
     "tslib": "catalog:",
-    "type-fest": "^4.40.1",
+    "type-fest": "^4.41.0",
     "typescript": "catalog:",
+    "typescript-eslint": "catalog:",
```

Comment:
what's the difference between this package and `@typescript-eslint/eslint-plugin`?